pkgs/development/tools/misc/tokei/default.nix

```diff
@@ -1,27 +1,52 @@
-{ stdenv, fetchFromGitHub, rustPlatform, libiconv, darwin }:
+{ lib, stdenv, fetchFromGitHub, rustPlatform, libiconv, darwin
+
+# feature flags
+# the nixpkg package set looks up values for these flags from `config.tokei`.
+, includeJSONOutput ? false
```

Comment:
Is there a reason to make these features optional?  Typically we build packages with all features enabled by default.

Reply:
They're not default on tokei's end. I don't really know why you wouldn't want them; they do introduce new dependencies. I did notice `serde_json` is in the dependency tree even without the feature on, though I don't know if it's the same version or if the feature compiles it twice.

I guess I should do a compilation time comparison between the two, and unless it's atrocious, then I can remove the flags and just default them to on because these don't pull in external dependencies or anything.

Reply:
Release build with all the features takes about 14% more time, in my quick test (building directly rather than through Nix, but w/e). And that's only 2m02s on this machine. I'll just enable them all.

Reply:
PR updated to remove these flags and just enable all features.